Makefile.am

```diff
@@ -23,7 +23,7 @@ AM_LDFLAGS =
 ##
 #
 
-LIBMTBL_VERSION_INFO=3:1:2
+LIBMTBL_VERSION_INFO=4:0:0
```

Comment:
I believe this should be incremented to `3:2:2`, as the varint refactoring and other speedups did not change the ABI, and `mtbl_iter_seek` was in the ABI before, and only restored to the header file (which isn't an ABI change).

4:0:0 would be appropriate for a backwards incompatible ABI change, such as removing or notably changing the semantics of an existing function. I don't believe there's any of that in this release.

Reply:
Okay, thanks for that! I'm still learning about libtool versioning vs regular semantic versioning. I'll make this change now.